dotCMS/src/main/java/com/dotmarketing/cms/urlmap/filters/URLMapFilter.java

```diff
@@ -142,12 +143,8 @@ public void doFilter(ServletRequest req, ServletResponse res, FilterChain chain)
             return;
         }
         if (RegEX.contains(url, mastRegEx)) {
-            boolean ADMIN_MODE = (optSession != null
-                    && optSession.getAttribute(com.dotmarketing.util.WebKeys.ADMIN_MODE_SESSION)
-                    != null);
-            boolean EDIT_MODE = ((optSession != null
-                    && optSession.getAttribute(com.dotmarketing.util.WebKeys.EDIT_MODE_SESSION)
-                    != null) && ADMIN_MODE);
+            boolean ADMIN_MODE =   PageMode.get(request) .isAdmin;
+            boolean EDIT_MODE =   PageMode.get(request)  == PageMode.EDIT_MODE;
```

Comment:
![MINOR](https://sonarsource.github.io/sonar-github/severity-minor.png 'Severity: MINOR') Rename this local variable to match the regular expression '^[a-z][a-zA-Z0-9]*$'. [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=squid%3AS00117)